templates/es6/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Entrypoint to application for es6 support
+ */
+ 
+require("babel-core/register");
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app').default;
```

Comment:
Nothing in this files seems to be in the ES6 style. Should it be updated to fit in with the rest of the files?